packages/react-native-editor/ios/GutenbergDemo/GutenbergViewController.swift

```diff
@@ -286,7 +286,7 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
             .unsupportedBlockEditor: unsupportedBlockEnabled,
             .canEnableUnsupportedBlockEditor: unsupportedBlockCanBeActivated,
             .mediaFilesCollectionBlock: true,
-            .audioBlock: true,
+			.isAudioBlockMediaUploadEnabled: true,
```

Comment:
Similar to my previous comment, looks like this line has a longer indentation than the rest.

Reply:
This [commit](https://github.com/WordPress/gutenberg/pull/31966/commits/11a79e7cd70695b06ddaf0dcf81a63defd2066b7) addressed the issue.